src/App.tsx

```diff
@@ -15,71 +16,85 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+interface State {
+  selectedGood: string,
+}
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends React.Component<{}, State> {
+  state: Readonly<State> = { selectedGood: 'Jam' };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  clearGoodsSelection = () => {
+    this.setState({ selectedGood: '' });
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  render() {
+    const { selectedGood } = this.state;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+    return (
+      <main className="section container">
+        <h1 className={`title ${selectedGood ? 'is-flex is-align-items-center' : ''}`}>
+          <div>
+            {selectedGood ? (
+              <div>
+                {`${selectedGood} is selected`}
+                {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+                <button
+                  data-cy="ClearButton"
+                  type="button"
+                  className="delete ml-3"
+                  onClick={this.clearGoodsSelection}
+                />
+              </div>
+            ) : (
+              <div>No goods selected</div>
+            )}
+          </div>
+        </h1>
+        <table className="table">
+          <tbody>
+            {goods.map(good => {
+              const goodIsSelected = good === selectedGood;
```

Comment:
```suggestion
              const isGoodSelected = good === selectedGood;
```
![image](https://user-images.githubusercontent.com/96409563/224819955-9f7452d0-0453-4d3a-8d6c-142f78f25bb5.png)